.github/workflows/Build_all.yml

```diff
@@ -178,11 +178,11 @@ jobs:
         export FC=mpif90
```

Comment:
also `rm -rf ./build`?

Reply:
For the first build of the step, no cleaning is needed